clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/view/EcsApplicationProviderSpec.groovy

```diff
@@ -61,8 +61,8 @@ class EcsApplicationProviderSpec extends Specification {
       desiredCount: 1,
       createdAt: new Date()
     )
-    def attributes = ServiceCachingAgent.convertServiceToAttributes(credentials.getName(),
-      credentials.getRegions()[0].getName(), service)
+    def attributes = TestServiceCachingAgent.create(credentials,
+      credentials.getRegions()[0].getName()).convertServiceToAttributes(service)
```

Comment:
The `convertServiceToAttributes()` method used to be a static method on the `ServiceCachingAgent` class. As we need to pass in the naming strategy to be able to derive the correct moniker values, it can no longer be static.

This meant that the tests needed a way to create service caching agent instances for testing and hence the creation of the test caching agent.